.github/workflows/RcppDeepState.yaml

```diff
@@ -20,4 +17,8 @@ jobs:
 
       - uses: FabrizioSandri/RcppDeepState-action@main
         with:
-          comment: 'true'
\ No newline at end of file
+          comment: 'true'
+          max_inputs: 10
+	  seed: 5
```

Comment:
RcppDeepState-action has not been executed because of this wrong indentation of the workflow file.